common/src/main/java/com/wynntils/models/character/SkillPointModel.java

```diff
@@ -126,6 +126,7 @@ public void deleteLoadout(String name) {
     }
 
     public void loadLoadout(String name) {
+        if (name == null) return; // no loadout selected
```

Comment:
Is this even clickable, or rather, callable, if name is not null?

Reply:
Oh yeah, I was exploring having loadouts be available by pressing enter, but this doesn't work so well with the name text box. Didn't remove this after.

Reply:
I think clicking the selected loadout for a 2nd time to load it could be nice if you do want an alternate method of loading